substra/sdk/backends/local/backend.py

```diff
@@ -426,7 +428,8 @@ def _add_task(self, key, spec, spec_options=None):
             inputs=_schemas_list_to_models_list(spec.inputs, models.InputRef),
             outputs=_output_from_spec(spec.outputs),
             tag=spec.tag or "",
-            status=models.Status.waiting,
+            # TODO: the waiting status should be more granular now
+            status=models.Status.waiting_for_builder_slot,
```

Comment:
Maybe we want `waiting_for_executor_slot` here, rather? 🤔

Reply:
Yes, I think so